### PR TITLE
chore: set version & add package.json volume mounts

### PR DESCRIPTION
**Pull request by Jonathansumner**
Changes to `docker-compose.yml` force `package.json` files containing versioning information to be mounted as a volume, keeping the version updated.
 - **Dependant on this [PR](https://github.com/fetchai/subql/pull/4)** to be merged for update

---

## Code Review Checklist (to be filled out by reviewer)

- [x] Description accurately reflects what changes are being made.
- [x] Either the PR references an issue (via the "Development" combobox) or the description explains the need for the changes.
- [x] The PR appropriately sized.
- [ ] ~~The PR contains an idempotent DB migration.~~
- [ ] ~~I have verified the correctness of the DB migration using relevant data (e.g. test-generated data).~~
- [ ] ~~New code has enough tests.~~
- [ ] ~~New code has enough documentation to answer "how do I use it?" and "what does it do?".~~
- [ ] Existing documentation is up-to-date, if impacted.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -38,6 +38,7 @@ services:
       CHAIN_ID: "fetchchain"
     volumes:
       - ./:/app
+      - ./docker/node-cosmos/package.json:/usr/local/lib/node_modules/@subql/node-cosmos/package.json
```
Comment (bryanchriswhite) [inline docker-compose.yml:41]: We should duplicate these changes for the ci-compose.yml and tracing-compose.yml files as well.

**Comment by bryanchriswhite** [comment]
https://github.com/fetchai/subql/pull/4 is approved! After that's merged could you please make sure you update the subql reference to the new `main` - then I think this is good to go! :smile:

**Comment by bryanchriswhite** [comment]
Maybe we could add a short bit to the introduction document about 1) the version fields, 2) the fact that `_metadata` has more in it, and 3) where to look for more info (graphql schema and/or source code snippets / links would be sufficient in absence of relevant docs).

File: project.yaml [hunk 2]
```diff
@@ -14,8 +14,8 @@ repository: https://github.com/fetchai/ledger-subquery
 schema:
   file: ./schema.graphql
 network:
-  chainId: dorado-1
-  endpoint: https://rpc-dorado.fetch.ai:443
+  chainId: fetchchain
+  endpoint: http://fetch-node:26657
```
Comment (bryanchriswhite) [inline project.yaml:18]: I don't think you intended to commit project.yaml or subql.

File: docs/introduction.md [hunk 3]
```diff
@@ -193,3 +193,55 @@ _(see: [schema.graphql](https://github.com/ledger-subquery/blob/main/schema.grap
...
+```graphql
+query ReleaseVersionTest {
+  _metadata {
+    queryNodeVersion
+    indexerNodeVersion
+  }
+}
+```
+
+Each of these version numbers are stored as the value to the key `"version"` within their relevant module `package.json` file. These files can be found in the `docker/node-cosmos/node_modules` and `subql/packages/query/` directories for the Indexer and GraphQL versions, respectively.
```
Comment (bryanchriswhite) [inline docs/introduction.md:211]:
```suggestion
Each of these version numbers are stored as the value to the key `"version"` within their relevant module `package.json` file. These files can be found in the `docker/node-cosmos/` and `subql/packages/query/` directories for the Indexer and GraphQL versions, respectively.
```

**Review by bryanchriswhite — CHANGES_REQUESTED** [review]
Thanks @Jonathansumner - couple of quick fixes and I think we're good to go!

File: docs/introduction.md (hunk 3 above)
Comment (Jonathansumner) [reply docs/introduction.md:211]: In my local repo the `package.json` path is "docker/node-cosmos/package.json" - am I meant to be referencing the `@suql` directory in docker/node-cosmos/node-modules`?

**Review by bryanchriswhite — APPROVED** [review]
Thanks @Jonathansumner!

**Pull request merged**